textfield/harness.ts

```diff
@@ -8,6 +8,11 @@ import {Harness} from '../testing/harness.js';
 
 import {TextField} from './internal/text-field.js';
 
+export const inputEventInitBase: InputEventInit = {
```

Comment:
Can you inline this where it's used rather than creating a shared variable? It'll make the tests a bit more damp and help with scanning code to see what it's doing

Reply:
Sure!